README.md

```diff
@@ -1,2 +1,124 @@
-# technical-documentation-page
-###this repository is about making documentation page.
+# technical-documentation-page 🚀
+
+## Project Description 📝
+
+> this project is about making technical documentation page. it has showed HTML coding language, is uesful for web development students, included some specific information about HTML.
+
+Examples:
+
+- working with HTML flie inspired me to create this documentation.
+- this will solve both theoreical and practical problems in the level of intermediate.
+- during workung on this documentation i learned more ideas about how creat technical documentation through HTML and CSS languages ,which codes i can use, how organize information together.
+- using of HTML and CSS languages make this documentation stand out.
+
+```html
+<!-- <nav id="navbar">
+      <header>HTML Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction_HTML">Introduction HTML</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Basic_Examples">HTML Basic Examples</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Styles">HTML Styles</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Lists">HTML Lists</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Atrributes">HTML Atrributes</a>
+        </li>
+      </ul>
+    </nav>Include code snippets or examples to demonstrate your project. -->
+```
+
+```css
+/* body {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 20px;
+  background-image: url(images1.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+}You can also include CSS code snippets. */
+```
+
+
+## Demo 📸
+
+Include a demo or animated GIF of your project. You can use tools like [asciinema](https://asciinema.org/) to record your terminal sessions.
+
+![Uploading documentation shot.jpg…](demo)
```

Comment:
- This one is not working, it's not showing your project image here. Try to fix it.